app/src/main/java/com/github/freeman/bootcamp/GameOptionsActivity.kt

```diff
@@ -49,7 +49,7 @@ import java.util.*
 
 class GameOptionsActivity : ComponentActivity() {
 
-    private val gameId = UUID.randomUUID().toString()
+    private val gameId = "TestGameId"//UUID.randomUUID().toString()
```

Comment:
I understant that you chose to use "testGameId" for now, since the guessing activity doesn't yet have another activity that launches it while passing a game ID. 
I would however suggest to put this string in a variable in the strings.xml file, because you use it more than once, which will make modifying or removing it easier.

Reply:
I saw your comment just after pressing the "merge" button so I will change that with my last task